src/d3a/d3a_core/sim_results/device_statistics.py

```diff
@@ -1,7 +1,8 @@
 from typing import Dict
 from d3a import limit_float_precision
 from d3a_interface.utils import create_or_update_subdict
-
+from d3a.d3a_core.sim_results import is_load_node_type, is_cell_tower_type, is_pv_node_type, \
+    is_prosumer_node_type
 FILL_VALUE = None
```

Comment:
Trivial, but I am wondering why flake does not complain. I was under the impression one empty line was needed here. Nevermind, if flake does not complain all is well.

Reply:
No, it didn't weirdly.